signer/src/bitcoin/validation.rs

```diff
@@ -134,14 +135,20 @@ impl BitcoinPreSignRequest {
     {
         let mut cache = ValidationCache::default();
 
+        let bitcoin_chain_tip = &btc_ctx.chain_tip;
+        let stacks_chain_tip = db.get_stacks_chain_tip(bitcoin_chain_tip).await?;
+        let Some(stacks_chain_tip) = stacks_chain_tip.map(|b| b.block_hash) else {
```

Comment:
It's a little confusing to reuse `stacks_chain_tip` for both the option and the wrapped value.  Maybe call the option `maybe_stacks_chain_tip`?

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/1344/commits/7f2bcddfa4e6884d1f116ed2ac778a1eda25993a.